### PR TITLE
Clean guardian_authorship

### PR DESCRIPTION
**Pull request by yongzx**
Tasks:
- [X] add original tasks (≥ 5 original tasks)
- [X] add metrics
- [X] add answer choices to the field and into the prompts.
- [X] indicate answer choices in prompt if exists

Discussion:
~~Does it make sense to prompt without answer choices in the prompt? For instance, see prompt "who_wrote_article".~~ 

Edit: Reread the instructions and found my answer from instruction step 11.

### CODE REVIEW

File: promptsource/templates/guardian_authorship/cross_genre_1/templates.yaml [hunk 1]
```diff
@@ -22,12 +28,14 @@ templates:
       ,\n  \"georgemonbiot\",\n  \"hugoyoung\",\n  \"jonathanfreedland\",\n  \"martinkettle\"\
       ,\n  \"maryriddell\",\n  \"nickcohen\",\n  \"peterpreston\",\n  \"pollytoynbee\"\
       ,\n  \"royhattersley\",\n  \"simonhoggart\",\n  \"willhutton\",\n  \"zoewilliams\"\
-      \n] [author]\n}}  on the topic of {{\n[\n  \"Politics\",\n  \"Society\",\n \
-      \ \"UK\",\n  \"World\",\n  \"Books\"\n] [topic]\n}} |||\n{{article}}"
+      \n] [author].capitalize()\n}}  on the topic of {{\n[\n  \"Politics\",\n  \"\
```
Comment (VictorSanh) [inline promptsource/templates/guardian_authorship/cross_genre_1/templates.yaml:31]:
```suggestion
      \n] [author].capitalize()\n}} on the topic of {{\n[\n  \"Politics\",\n  \"\
```

File: promptsource/templates/guardian_authorship/cross_genre_1/templates.yaml [hunk 2]
```diff
@@ -36,49 +44,133 @@ templates:
...
+    reference: ''
   68bfa6a4-a89c-4be2-aa0b-cce1103e3ecf: !Template
-    answer_choices: null
+    answer_choices: Politics ||| Society ||| UK ||| World ||| Books
     id: 68bfa6a4-a89c-4be2-aa0b-cce1103e3ecf
-    jinja: "What is the topic of this article?\n\n{{article}} |||\n{{\n[\n  \"Politics\"\
-      ,\n  \"Society\",\n  \"UK\",\n  \"World\",\n  \"Books\"\n][topic]\n}}"
+    jinja: "What is the topic of this article? The answer options are {% for topic\
+      \ in answer_choices[:-1] %} {{topic.capitalize()}}, {% endfor %} {{answer_choices[-1].capitalize()}}.\n\
+      \n{{article}} \n|||\n{{answer_choices[topic]}}"
```
Comment (VictorSanh) [inline promptsource/templates/guardian_authorship/cross_genre_1/templates.yaml:82]:
```suggestion
    jinja: "What is the topic of this article? The answer options are {{answer_choices|join(\"\
      , \")}}.\n\n{{article}} \n|||\n{{answer_choices[topic]}}"
```
simpler way to write it

Comment (VictorSanh) [reply promptsource/templates/guardian_authorship/cross_genre_1/templates.yaml:82]: Can be applied to a lot of other places

File: promptsource/templates/guardian_authorship/cross_genre_1/templates.yaml [hunk 3]
```diff
@@ -2,17 +2,23 @@ dataset: guardian_authorship
 subset: cross_genre_1
 templates:
   026e1ef2-c765-4262-b7b3-a087f38907db: !Template
-    answer_choices: null
+    answer_choices: catherinebennett ||| georgemonbiot ||| hugoyoung ||| jonathanfreedland
+      ||| martinkettle ||| maryriddell ||| nickcohen ||| peterpreston ||| pollytoynbee
+      ||| royhattersley ||| simonhoggart ||| willhutton ||| zoewilliams
```
Comment (VictorSanh) [inline promptsource/templates/guardian_authorship/cross_genre_1/templates.yaml:7]:
```suggestion
    answer_choices: Catherine Bennett ||| George Monbiot ||| Hugo Young ||| Jonathan Freedland
      ||| Martin Kettle ||| Mary Riddell ||| Nick Cohen ||| Peter Preston ||| Polly Toynbee
      ||| Roy Hattersley ||| Simon Hoggart ||| Will Hutton ||| Zoe Williams
```

Comment (VictorSanh) [reply promptsource/templates/guardian_authorship/cross_genre_1/templates.yaml:7]: same for other places

File: promptsource/templates/guardian_authorship/cross_genre_1/templates.yaml [hunk 4]
```diff
@@ -36,49 +44,133 @@ templates:
...
+      }}, below.
+
+
+      {{article}}
+
+      |||
+
+      {{ answer_choices[author].capitalize() }}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
```
Comment (VictorSanh) [inline promptsource/templates/guardian_authorship/cross_genre_1/templates.yaml:71]:
```suggestion
      choices_in_prompt: true
```

**Review by VictorSanh — CHANGES_REQUESTED** [review]
thanks! mostly good to me.
A few changes requested:
- Don't use capitalize, directly capitalize inside the answer_choices
- Separate the names and family names in the answer_choices and in the lists of journalist names in the templates
- If you are not using "and" to list (i.e. A, B, C rather than A, B and C), you can use a simpler writing: {{answer_choices|join(", ")}}

**Comment by yongzx** [comment]
@VictorSanh Fixed the following:
- Answer choices (capitalize the author names and make them more natural)
- Use join instead of for-loop.
- Indicate answer choices exist in the templates.

**Comment by VictorSanh** [comment]
thank you @yongzx , lgtm! merging now, i updated the spreadsheet too

**Pull request merged**